coreneuron/io/nrn_filehandler.cpp

```diff
@@ -27,7 +27,7 @@ THE POSSIBILITY OF SUCH DAMAGE.
 */
 
 #include <iostream>
-#include "coreneuron/nrniv/nrn_filehandler.h"
+#include "nrn_filehandler.h"
```

Comment:
Most of the header files have relative include to top level (`coreneuron/xxx`). Any reason for these to not following the same pattern?

Reply:
humm.. you know, I will have to go through this slowly. clion did the refactoring but did the includes like this when the two files were in the same folder. I'll clean it up. Thanks for noticing.